htop.1.in

```diff
@@ -67,6 +67,11 @@ requesting a sort order with -s.
 \fB\-H \-\-highlight-changes=DELAY\fR
 Highlight new and old processes
 .TP
+\fB   \-\-kiosk\fR
+Start htop in kiosk-mode.
```

Comment:
htop should be in bold here, i.e.

```
Start
.B htop
in kiosk-mode, which hides process selection and function bar.
```

Reply:
Or skip `htop` from that sentence.

Reply:
@natoscott added an explanation, that is important.
Also add to the manual that it implies `--readonly` as stated in the built-in `--help`.